### DISCUSSION TITLE
GitHub Actions support

### DISCUSSION THREAD

**Discussion by KaKi87** [marcosnils/bin#213] [Ideas]
Hi,

Some developers don't create releases at all but only allow downloading via GitHub Actions artifacts, e.g. [Abaddon](https://github.com/uowuo/abaddon).

It would be nice if `bin` could handle that.

Thanks

**Comment by marcosnils** [comment]
this is a cool idea indeed. TIL  about nighly.link.  `bin` should be able to handle this, I think it might become part of #179

**Reply by KaKi87** [reply]
Oh, I didn't know [nightly.link](https://nightly.link) either, thank you for the discovery ! I thought there were gonna be a need to use the API.
